### ISSUE TITLE
Toolbar pattern: improve accessibility of up and down arrows

### ISSUE THREAD

**Issue by erral** [plone/mockup#959]
Plone toolbar shows up and down arrows when there are multiple items in the menu, the accessibility of those arrows could be improved taking the following steps:

- Those arrows links lack any title.
- The up arrow is useless when the topmost item of the menu is shown, and does nothing if you click on it.
- The down arrow is useless when the last item of the menu is shown, and does nothing if you click on it.
- Those links are useless for users using screen readers or those browsing with the TAB keyboard, because the toolbar scrolls automatically. 

@ionlizarazu
